packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -273,6 +287,38 @@ export class SingleAddressWallet implements ObservableWallet {
     return this.name;
   }
 
+  async initializeVotingRegistrationTx(props: InitializeVotingTxProps): Promise<InitializeVotingTxResult> {
+    const auxiliaryData = await this.#constructVotingAuxData({
+      votingPublicKey: props.votingPublicKey,
+      networkId: props.networkId,
+      nonce: props.nonce,
+    });
+    const ownAddress = (await firstValueFrom(this.addresses$))[0].address;
+    // Set min output amount to perform voting registration transaction. Coins will be sent to wallet (owner) address
+    const outputs = new Set([{
+      address: ownAddress,
+      value: { coins: 1_000_000n }
+    }]);
+    const { constraints, utxo, implicitCoin, validityInterval, changeAddress } = await this.#prepareTx({
```

Comment:
Why not just use `this.initializeTx`? If we can do that then entire `initializeVotingRegistrationTx` is not very useful and it would be enough to export a util to `constructVotingAuxData`.